.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I would guess that one can reformulate all enhancements as *small* feature requests. Thus, for the sake of simplicity, we could drop the enhancement template. Not a strong opinion.

Reply:
I thought about that as well. The reason why I decided to add it is because of issues like #2373, #2484 or #2393, which aren't really (small) features, but instead improvements/ changes in exiting features. 
But not a strong opinion on my side either. Could also rename it to _improvement_ / _change / ...?,_ but _enhancement_ is the common term in other repos.

Reply:
:+1: Let's keep it and see how often it is used.